app/models/capital.rb

```diff
@@ -1,7 +1,8 @@
 class Capital < ApplicationRecord
-  attribute :premium_bonds_total_value, :pence
-
   belongs_to :crime_application
+  attribute :premium_bonds_total_value, :pence
+  attribute :trust_fund_amount_held, :pence
+  attribute :yearly_dividend, :pence
```

Comment:
I wonder if this attribute needs to be trust_fund_yearly_dividend as it is stored on capital and other things could have a yearly_dividend?